config/migrations/2023/lpdc/20230512123030-add-concept-display-configurations.sparql

```diff
@@ -0,0 +1,42 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX cpsv: <http://purl.org/vocab/cpsv#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX lpdcExt: <https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX pav: <http://purl.org/pav/>
+
+INSERT {
+  GRAPH ?graph {
+    ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri .
+    ?configUri a lpdcExt:ConceptDisplayConfiguration ;
+      mu:uuid ?configId ;
+      lpdcExt:conceptIsNew false ; # TODO, do we want to set this to true? That might cause a _lot_ of "nieuw" labels to appear?
+      lpdcExt:conceptInstantiated false ;
+      dct:relation ?eenheid .
+  }
+}
+WHERE {
+  VALUES ?eenheidClassificatie {
+    <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001> # gemeente
+  }
+
+  ?eenheid a besluit:Bestuurseenheid;
+    mu:uuid ?eenheidId;
+    besluit:classificatie ?eenheidClassificatie.
+
+  BIND(IRI(CONCAT("http://mu.semte.ch/graphs/organizations/", STR(?eenheidId), "/LoketLB-LPDCGebruiker")) as ?graph)
+
+  # find the concepts which have no config yet
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?concept a lpdcExt:ConceptualPublicService
+  }
+
+  FILTER NOT EXISTS {
+    GRAPH ?graph {
+      ?concept lpdcExt:hasConceptDisplayConfiguration ?configUri
+    }
+  }
```

Comment:
Something is still missing here I think. It also adds new ConceptDisplayConfigurations even if one exists for that concept / bestuurseenheid combo already.

Reply:
I went for a different solution which sidesteps this issue.